app/src/main/res/values/styles.xml

```diff
@@ -73,6 +73,10 @@
         <item name="historyURLColor">@color/history_url_normal_theme</item>
         <item name="historyHeader">@color/history_header_normal_theme</item>
 
+        <!-- Bookmark fragment colors -->
+        <item name="bookmarksLabelColor">@color/bookmarks_label_normal_theme</item>
```

Comment:
I think these colors @color/bookmarks_label_normal_theme and @color/bookmarks_edit_normal_theme should also be overloaded in values-night/colors/xml for dark theme